pkg/koordlet/metricsadvisor/collectors/coldmemoryresource/cold_page_kidled.go

```diff
@@ -0,0 +1,238 @@
+/*
+Copyright 2022 The Koordinator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package coldmemoryresource
+
+import (
+	"fmt"
+	"time"
+
+	"path/filepath"
+
+	"go.uber.org/atomic"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/klog/v2"
+
+	"github.com/koordinator-sh/koordinator/pkg/koordlet/metriccache"
+	"github.com/koordinator-sh/koordinator/pkg/koordlet/metricsadvisor/framework"
+	"github.com/koordinator-sh/koordinator/pkg/koordlet/resourceexecutor"
+	"github.com/koordinator-sh/koordinator/pkg/koordlet/statesinformer"
+	koordletutil "github.com/koordinator-sh/koordinator/pkg/koordlet/util"
+	"github.com/koordinator-sh/koordinator/pkg/koordlet/util/system"
+	"github.com/koordinator-sh/koordinator/pkg/util"
+)
+
+type kidledcoldPageCollector struct {
+	collectInterval time.Duration
+	started         *atomic.Bool
+	cgroupReader    resourceexecutor.CgroupReader
+	statesInformer  statesinformer.StatesInformer
+	podFilter       framework.PodFilter
+	appendableDB    metriccache.Appendable
+	metricDB        metriccache.MetricCache
+}
+
+func (k *kidledcoldPageCollector) Run(stopCh <-chan struct{}) {
+	go wait.Until(k.collectColdPageInfo, k.collectInterval, stopCh)
+}
+func (k *kidledcoldPageCollector) Started() bool {
+	return k.started.Load()
+}
+func (k *kidledcoldPageCollector) Enabled() bool {
+	return true
+}
+func (k *kidledcoldPageCollector) Setup(c1 *framework.Context) {}
+func (k *kidledcoldPageCollector) collectColdPageInfo() {
+	if k.statesInformer == nil {
+		return
+	}
+	klog.V(4).Info("collectColdPageInfo start")
```

Comment:
this kind of log is normal and repeating in each turn, so it should be set as higher verbosity